README.md

```diff
@@ -147,6 +147,7 @@ Note that as part of the onboarding process your repository name may be normaliz
 [tests-nullish-coalescing]: https://github.com/tc39/test262/pull/2402
 [promise-any]: https://github.com/tc39/proposal-promise-any
 [promise-any-notes]: https://github.com/tc39/tc39-notes/blob/master/meetings/2019-07/july-24.md#promiseany
+[tests-promise-any]: https://github.com/tc39/test262/issues/2410
```

Comment:
```suggestion
[tests-promise-any]: https://github.com/tc39/proposal-promise-any
```

Reply:
@JakeChampion thank you for you suggestion. This should be a link to test262 repo for tracking tests status, so the current URL is correct